tests/console/gdb.pm

```diff
@@ -15,12 +15,12 @@ use warnings;
 use testapi;
 use utils 'zypper_call';
 
-sub wait_serial_or_croak {
+sub wait_serial_or_die {
     my $feedback = shift;
 
     my $e = wait_serial($feedback, 10);
     if (!defined $e) {
-        croak("Unexpected serial output");
+        die("Unexpected serial output");
```

Comment:
Please use croak instead, since a `die` will mention always L23 whereas croak would mention the line where the current caller comes from (e.g L38), ` use Carp 'croak'` if you want to be minimalistic

Reply:
It actually mentions `Test died: Unexpected serial output at /usr/lib/os-autoinst/autotest.pm line 365.`

Reply:
http://10.161.229.249/tests/211#step/gdb/2

Reply:
Well, let's stay with `die` then